zerver/webhooks/gocd/view.py

```diff
@@ -9,13 +9,26 @@
 from zerver.lib.typed_endpoint import JsonBodyPayload, typed_endpoint
 from zerver.lib.validator import WildValue, check_string
 from zerver.lib.webhooks.common import check_send_webhook_message
+from zerver.lib.webhooks.git import get_short_sha
 from zerver.models import UserProfile
 
-MESSAGE_TEMPLATE = """\
-Author: {}
-Build status: {} {}
-Details: [build log]({})
-Comment: {}"""
+COMMIT_INFO_TEMPLATE = (
+    """Triggered on [`{commit_details}`]({commit_link}) on branch `{branch_name}`."""
```

Comment:
Looking at the message I feel like, "**Commit**: `commit-sha-link` on branch `name`" would be fine. I'm not sure what's gained by "Triggered on"? I'd also drop the period here.

Reply:
hmm yeah, that'd probably be fine.